packages/emails/src/components/CallToAction.tsx

```diff
@@ -1,10 +1,11 @@
 import { LinkIcon } from "./LinkIcon";
 
-export const CallToAction = (props: { label: string; href: string }) => (
+export const CallToAction = (props: { label: string; href: string; secondary?: boolean }) => (
   <p
     style={{
       display: "inline-block",
-      background: "#292929",
+      background: props.secondary ? "#FFFFFF" : "#292929",
+      border: props.secondary ? "1px solid #292929" : "",
```

Comment:
@Jaibles are you happy with those styles?

Reply:
![CleanShot 2022-12-08 at 23 21 44](https://user-images.githubusercontent.com/8019099/206579686-d03adba6-a0ed-4044-9b48-2c7a3f2984e5.jpg)

Reply:
The styles should match our design system. It doesn't look like they do.
<img width="572" alt="CleanShot 2022-12-09 at 16 11 07@2x" src="https://user-images.githubusercontent.com/4536123/206744442-577ccaba-aa64-4787-950e-cd7baa3dda56.png">
https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=309%3A42062&t=BvUyxGFdLgVige8O-1

Reply:
Yeah, I don't think our buttons in emails were upgraded to v2, I can work on it, thanks @Jaibles @PeerRich